### DISCUSSION TITLE
Installation error

### DISCUSSION THREAD

**Discussion by FrenkAnd** [AdguardTeam/AdGuardHome#3989] [Q&A]
When installing the web interface. Error Error: control/install/check_config | Network Error. How to fix it ?

**Comment by EugeneOne1** [comment]
@FrenkAnd, actually should be fixed in #3987.  Try latest `edge` build, see the [instructions](https://github.com/AdguardTeam/AdGuardHome#test-unstable-versions) on how to install it, if you're not familiar with the process yet.  It will also be available **v0.107.1** rather soon.
